_data/projects.yml

```diff
@@ -315,6 +315,12 @@
   code: "https://github.com/AndrewAnnex/SpiceyPy"
   contact: "Andrew M. Annex"
   keywords: ["solar","magnetosphere","ionosphere_thermosphere_mesosphere","general"]
+  community: ["https://img.shields.io/badge/Good-brightgreen.svg", "Good"]
```

Comment:
Does spiceypy have contribution guidelines anywhere? It was one of the things I noticed HelioPy is missing when I did its review.

Reply:
Yes there is a contributing.md file in the repo at https://github.com/AndrewAnnex/SpiceyPy/blob/master/CONTRIBUTING.md

Reply:
Ah cool, I was hunting round the docs!

Reply:
could the grading template be updated or the meta data such that it could include links to the docs, contributing guide, etc?

Reply:
Hey @AndrewAnnex - good thought. I'll pass that suggestion along to the new PyHC developer, Tom Baltzer, for him to potentially implement!

Reply:
In the mean time, though, gonna merge your PR so your grading shows up.